resources/measures/ChangeBuildingLocation/measure.rb

```diff
@@ -358,6 +364,62 @@ def run(model, runner, user_arguments)
       runner.registerInfo("Setting Climate Zone to #{climateZones.getClimateZones('ASHRAE').first.value}")
     end
 
+    # set soil properties as building additional properties for ground source heat pump modeling
+    soil_conductivity = runner.getDoubleArgumentValue('soil_conductivity', user_arguments)
+
+    # get climate zone and set undisturbed ground temp (used for ground source heat pump modeling)
+    climate_zone = runner.getStringArgumentValue('climate_zone', user_arguments)
+    puts "climate zone = #{climate_zone}"
+
+    undisturbed_ground_temps = {
```

Comment:
move this to the measure resources directory, and organize by institution. Ordering by ground temp is confusing.